nvtabular/dask/io.py

```diff
@@ -177,10 +180,11 @@ def __init__(
         storage_options=None,
         **kwargs,
     ):
-
+        self.kwargs = kwargs
         if part_size:
             # If a specific partition size is given, use it directly
-            part_size = parse_bytes(part_size)
+            # part_size = parse_bytes(part_size)
+            part_mem_fraction = part_size
```

Comment:
I know this might be a temporary change for debugging, but I want to make sure it is clear that `part_size` is different from `part_mem_fraction`.  It is the number of bytes the user wants for a partition size, and could be in passed in like: `part_size="1GB"` (hence the need for `parse_bytes`)

Reply:
oh!!! Nice, yeah, that parse bytes turns  0.1 0.01 into 0 which is not the desired action... So then to explicitly say I want to use a certain GPU fraction... I have to know how much actual memory that is? (like "24GB") or something like that.

Reply:
> So then to explicitly say I want to use a certain GPU fraction... I have to know how much actual memory that is? (like "24GB") or something like that.

Yes - the default behavior is that `part_size` is not provided, and we will use a user-specified or default value for `part_mem_fraction` to figure out what `part_size` *should* be.  However, if the user already knows the specific size they want, they should be able to pass it directly with `part_size`.

Reply:
Feel free to suggest a better description if the docstring is confusing for `part_size`

Reply:
Ahh ok nice, I was using the wrong arg then. I needed to focus my attention on part_mem_fraction. Thanks for clearing that up